docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+services:
+  app:
+    depends_on:
+      - mongodb
+    build: .
+    restart: unless-stopped
+    environment:
+      # You can leave this as-is.
+      - DB_HOST=mongodb
+      - DB_USER=mongouser
+      - DB_PASSWORD=mongopasword
+      - DB_NAME=xyter
+      - DB_PORT=27017
+      # You have to set these values!
+      - DISCORD_TOKEN=YOUR_DISCORD_TOKEN
+      - DISCORD_CLIENT_ID=YOUR_CLIENT_ID
+      - DISCORD_DEV_GUILD_ID=YOUR_DEV_GUILD_ID
+      - HOSTER_NAME=YOUR_HOSTER_NAME
+      - NODE_ENV=development
+    stdin_open: true
+    tty: true
```

Comment:
Could you maybe also make the ability to pass troguh logs from root directory/logs to like /logs as a docker volume?